scripts/storage-slots.sh

```diff
@@ -0,0 +1,26 @@
+# Scripts for determining storage slots, useful for tests
+
+token=0xd632f22692FaC7611d2AA1C0D552930D43CAEd3B # fraxCrv
+holder=0x99780beadd209cc3c7282536883ef58f4ff4e52f
+
+# balanceOf
+# for slot in {0..250}; do
```

Comment:
Just curious why all the code is commented out? I think it'd be good to commit a working version that you can run without modifications

Reply:
I commented it out because I figured the user could determine if they wanted either/both balanceOf or totalSupply storage slots and uncomment. But yeah agree, status quo should be uncommented and working seemlessly